waiter/integration/waiter/token_request_test.clj

```diff
@@ -662,6 +662,22 @@
       (is (not (str/includes? body "clojure")) body)
       (is (str/includes? body "The following environment variable keys are reserved: HOME.") body))))
 
+(deftest ^:parallel ^:integration-fast test-token-parameters-exceed-limits
+  (testing-using-waiter-url
+    (let [constraints (setting waiter-url [:service-description-constraints])
+          upper-limits (->> constraints
```

Comment:
I've seen it at least a couple of places.  Should we factor it out?

Reply:
Factored out in `extract-max-constraints`